packages/dev/core/src/Materials/Node/Blocks/reciprocalBlock.ts

```diff
@@ -19,7 +19,6 @@ export class ReciprocalBlock extends NodeMaterialBlock {
         this.registerOutput("output", NodeMaterialBlockConnectionPointTypes.BasedOnInput);
 
         this._outputs[0]._typeConnectionSource = this._inputs[0];
-        this._outputs[0].excludedConnectionPointTypes.push(NodeMaterialBlockConnectionPointTypes.Matrix);

```

Comment:
I wonder why the `Matrix` type has been explicitely excluded in the first place... Maybe there was a reason for that?

Reply:
I guess cause the function was different but let s ask @deltakosh to confirm ???